host-controller/src/main/java/org/jboss/as/host/controller/model/jvm/JVMAddHandler.java

```diff
@@ -43,12 +52,48 @@
         this.attrs = attrs;
     }
 
-    protected void populateModel(ModelNode operation, ModelNode model) throws OperationFailedException {
+    @Override
+    protected void populateModel(final OperationContext context, final ModelNode operation, final Resource resource) throws OperationFailedException {
+
+        super.populateModel(context, operation, resource);
+        final ModelNode model = resource.getModel();
+        final PathAddress jvm = PathAddress.pathAddress(operation.require(ModelDescriptionConstants.OP_ADDR));
+
+        context.addStep(new OperationStepHandler() {
+            public void execute(final OperationContext context, final ModelNode operation) throws OperationFailedException {
+                Resource root;
+                try {
+                    root = context.readResourceFromRoot(context.getCurrentAddress().getParent(), false);
+                } catch (Resource.NoSuchResourceException e) {
```

Comment:
This occurs in the domain ignore test case, where the slave is ignoring server-group="group2". When the group is added via changing the server group, the jvm child is added, but the parent isn't available at to RR. Is there a better way to do this?

It seems to work and test out fine, but it seems a bit clumsy.